etna/transforms/outliers/point_outliers.py

```diff
@@ -255,8 +258,147 @@ def params_to_tune(self) -> Dict[str, BaseDistribution]:
+            Whether to ignore missing values inside a series
+        n_estimators:
+            The number of base estimators in the ensemble
+        max_samples:
+            The number of samples to draw from X to train each base estimator
+                *  If int, then draw max_samples samples.
+
+                *  If float, then draw max_samples * X.shape[0] samples.
+
+                *  If “auto”, then max_samples=min(256, n_samples).
+
+            If max_samples is larger than the number of samples provided, all samples will be used for all trees (no sampling).
+        contamination:
+            The amount of contamination of the data set, i.e. the proportion of outliers in the data set.
+            Used when fitting to define the threshold on the scores of the samples.
+                *  If ‘auto’, the threshold is determined as in the original paper.
+
+                *  If float, the contamination should be in the range (0, 0.5].
+        max_features:
+            The number of features to draw from X to train each base estimator.
+                *  If int, then draw max_features features.
+
+                *  If float, then draw `max(1, int(max_features * n_features_in_))` features.
+            Note: using a float number less than 1.0 or integer less than number of features
+            will enable feature subsampling and leads to a longer runtime.
+        bootstrap:
+                *  If True, individual trees are fit on random subsets of the training data sampled with replacement.
+                *  If False, sampling without replacement is performed.
+        n_jobs:
+            The number of jobs to run in parallel for both fit and predict.
+                *  None means 1 unless in a joblib.parallel_backend context.
+                *  -1 means using all processors
+        random_state:
+            Controls the pseudo-randomness of the selection of the feature and split values for
+            each branching step and each tree in the forest.
+        verbose:
+            Controls the verbosity of the tree building process.
+
+        Notes
+        -----
+        To get more insights on parameters see documentation of Isolation Forest algorithm:
+
+        `Documentation for Isolation Forest <https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.IsolationForest.html>`_.
+        """
+        self.features_to_use = features_to_use
+        self.features_to_ignore = features_to_ignore
+        self.ignore_missing = ignore_missing
+        self.n_estimators = n_estimators
+        self.max_samples = max_samples
+        self.contamination = contamination
+        self.max_features = max_features
+        self.bootstrap = bootstrap
+        self.n_jobs = n_jobs
+        self.random_state = random_state
+        self.verbose = verbose
+        super().__init__(in_column=in_column, ignore_flag_column=ignore_flag_column)
+
+    def detect_outliers(self, ts: TSDataset) -> Dict[str, List[pd.Timestamp]]:
```

Comment:
Wrong typing here. `pd.Series` will be returned when `index_only=False`

Reply:
Fix typing everywhere